test/e2e_node/podresources_test.go

```diff
@@ -1027,8 +1027,8 @@ var _ = SIGDescribe("POD Resources", framework.WithSerial(), feature.PodResource
 					}
 					pod := makePodResourcesTestPod(pd)
 					pod.Spec.Containers[0].Command = []string{"sh", "-c", "/bin/true"}
-					pod = e2epod.NewPodClient(f).CreateSync(ctx, pod)
-					defer e2epod.NewPodClient(f).DeleteSync(ctx, pod.Name, metav1.DeleteOptions{}, time.Minute)
+					pod = e2epod.NewPodClient(f).Create(ctx, pod)
+					defer framework.ExpectNoError(e2epod.NewPodClient(f).Delete(ctx, pod.Name, metav1.DeleteOptions{}))
```

Comment:
OTOH we had multiple cases of pod "leaking" (= being deleted _after_ the owning test ended and a new test started) and causing flakes, so I'd rather keep the `DeleteSync` here. Or do we have a reason I'm missing to move to `Delete`?

Reply:
@ffromani Thank you for the review! I agree, using `DeleteSync` is safer.

Updated the PR, PTAL.